### ISSUE TITLE
2.3 release schedule

### ISSUE THREAD

**Issue by natemcmaster** [natemcmaster/CommandLineUtils#193]
Hey all, here are my plans for releasing 2.3 of this library.

* 2.3.0 - December 31
* 2.3-rc - December 14
* _2.3-beta - done https://github.com/natemcmaster/CommandLineUtils/releases/tag/v2.3.0-beta_
* _2.3-alpha - done https://github.com/natemcmaster/CommandLineUtils/releases/tag/v2.3.0-alpha_

We're almost out of time for 2.3, so please give the pre-releases a try and let me know if you run into issues.

Thanks,
Nate

**Comment by RonSkufca** [comment]
Nate, Are the pre-releases available as NuGet packages?

**Comment by natemcmaster** [comment]
Yes. See https://www.nuget.org/packages/McMaster.Extensions.CommandLineUtils/ and https://www.nuget.org/packages/McMaster.Extensions.Hosting.CommandLine

**Comment by RonSkufca** [comment]
Perfect thanks
